src/gen-ts.ts

```diff
@@ -99,6 +101,16 @@ function makeDeclarationsFilename(sourceUrl: string): string {
   return path.join(parsed.dir, parsed.name) + '.d.ts';
 }
 
+function makeHeader(sourceUrls: string[]): string {
+  return `DO NOT EDIT
+
+This file was automatically generated by
+  https://github.com/Polymer/gen-typescript-declarations
+
+Source files:
```

Comment:
I'd recommend:

`To change these typings you'll have to edit the source files:`

Reply:
Done.